GameProject/Assets/OpenWorld Assets/Scripts/ThirdPersonCameraController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ThirdPersonCameraController : MonoBehaviour
+{
+    [SerializeField] private Transform _lookAt;
+    [SerializeField] private Transform _camTransform;   // DELETE THIS
+
+
+    private Camera _cam;
+
+    private float _distance = 30.0f;    // DISTANCE TO STAY BEHIND PLAYER
```

Comment:
This might be better as a serializable, configurable value instead of just a hard 30.